### DISCUSSION TITLE
Is there a way to stop the player?

### DISCUSSION THREAD

**Discussion by headeer** [the-maazu/react-material-music-player#16] [Q&A] [answered]
How can I stop playing song from javascript?

**Comment by the-maazu** [comment, marked as answer]
You can do this by passing empty array to play().

I have considered exposing internal control commands in javascript but ultimately decided against since users already have full control via the UI.

But then again I understand some might want to hide player UI and implement custom controls.

Perhaps will expose all controls in the next release.

**Comment by the-maazu** [comment]
New functions have been added to the interface install with "package@beta" . read more [here](https://www.npmjs.com/package/react-material-music-player/v/1.2.0-beta.1)
